app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,31 @@
+class PostsController < ApplicationController
+  def index
+    
+    @post = Post.new
+    @posts = Post.all
+
+    
+  end
+
+  def create
+    
+
+    @post = Post.new
+    @posts = Post.all
```

Comment:
Not needed.
According to your UI, you can create a form on Post#index view and access to that method only when the user submit the post creation form